forum/settings/common.py

```diff
@@ -13,3 +13,14 @@ def plugin_settings(settings: Any) -> None:
     settings.FORUM_PORT = "4567"
     settings.MONGO_HOST = "mongodb"
     settings.MONGO_PORT = 27017
+
+    settings.FORUM_ELASTICSEARCH_INDEX_NAMES = ["comment_threads", "comments"]
+    settings.FORUM_MAX_DEEP_SEARCH_COMMENT_COUNT = 1000
+
+    settings.ELASTIC_SEARCH_CONFIG = [
```

Comment:
is this config already defined in LMS, can we check that. Alternatively, we can add `FORUM` prefix with this too.

Reply:
Yes, this setting is present in the LMS. I've copied this from the LMS.